django-backend/fecfiler/scha_transactions/serializers.py

```diff
@@ -80,12 +80,25 @@ class Meta:
 class SchATransactionParentSerializer(SchATransactionSerializer):
     children = ListSerializer(
         child=SchATransactionSerializer(),
-        write_only=True,
         allow_null=True,
         allow_empty=True,
         required=False,
     )
 
+    parent_transaction = SchATransactionSerializer(required=False, allow_null=True)
+
+    def to_representation(self, instance):
+        ret = super().to_representation(instance)
+
+        # Add child transactions to transaction.children field in JSON output
+        existing_children = SchATransaction.objects.filter(
+            parent_transaction_id=instance.id
+        ).all()
+        ret["children"] = map(
+            lambda child: SchATransactionParentSerializer(child).data, existing_children
```

Comment:
I think you can use self.to_representation(child) here for an exact match of functionality
`map(self.to_representation, existing_children)`

Reply:
Good tip. I made the change and it works.